control-plane/rest/openapi-specs/v0_api_spec.yaml

```diff
@@ -2369,6 +2401,28 @@ components:
       required:
         - snapshot
         - volume
+    ResizeVolumeBody:
+      example:
+        size: 104857610
+        capacity_limit: 204857610
+      description: Resize Volume Body
+      type: object
+      properties:
+        size:
+          description: Required new size of the volume in bytes
+          type: integer
+          format: uint64
+          minimum: 0
+        capacity_limit:
```

Comment:
We're not using this today, let's omit it and add it when we needed?

Reply:
faire enough. Removed for now. 
cc: @chriswldenyer